pkg/aws/ec2.go

```diff
@@ -48,7 +54,7 @@ type AssertEC2TagValueInput struct {
 	TagName string
 	// The value of the tag to assert.
 	Value string
-	// The Instance ID that the tag mustbe set on.
+	// The Instance ID that the method will assert has a tag with the specified tag name and the specified value.
```

Comment:
Ah, gotcha. No need to hold up this PR, but I think of this as...
```suggestion
	// The ID of the EC2 on which the tag is expected to be set.
```